contracts/governance/Vesting/fouryear/FourYearVestingFactory.sol

```diff
@@ -26,6 +26,7 @@ contract FourYearVestingFactory is IFourYearVestingFactory, Ownable {
 	 * @param _duration The total duration in seconds.
 	 * @param _feeSharing The address of fee sharing contract.
 	 * @param _vestingOwner The address of an owner of vesting contract.
+	 * @dev _vestingOwner should ALWAYS be multisig.
```

Comment:
renaming to `_vestingOwnerMultisig` will make it more clear that multisig should be used

Reply:
Done